test-resources/swift/cleanup_rules/input/SampleClass.swift

```diff
@@ -11,4 +11,8 @@
 
 class SampleClass {
     var isEnabled = TestEnum.stale_flag_one.isEnabled
+    var isEnabled2 = TestEnum.stale_flag_one.isEnabled && f1()
+    var isEnabled3 = TestEnum.stale_flag_one.isEnabled && v1
+    var isEnabled4 = f2() && TestEnum.stale_flag_one.isEnabled 
+    var isEnabled5 = v2 && TestEnum.stale_flag_one.isEnabled 
```

Comment:
```suggestion
    var isEnabled5 = v2 && TestEnum.stale_flag_one.isEnabled 
    var isEnabled6 = v2 && (v3 && TestEnum.stale_flag_one.isEnabled)
```
Let's handle the nested cases too

Reply:
Updated the query to handle nested cases, and added a Case `v2 && ( TestEnum.stale_flag_one.isEnabled && true)` since `v2 && (v3 && TestEnum.stale_flag_one.isEnabled)` would end up to `v2 && (v3)`

Reply:
SGTM. But can we simplify a paranthesized expression to get unparanthesized after simplification ?

Reply:
yes, we can do that in a separate rule